projects/memapGui/src/main/java/fortiss/gui/VolatileInputPanel.java

```diff
@@ -225,11 +238,18 @@ public void plot() {
 			plotPanel.setVisible(false);
 			plotPanel.setPlotted(false);
 		} else {
+			plotPanel.clearSeries();
 			Volatile volatileProducer = (Volatile) DesignerPanel.selectedComponent;
 			DataInterface data = volatileProducer.getData();
-			if (data != null) {	
+			if (data != null) {													
 				for (String seriesName : data.getSeriesList()) {
-					plotPanel.addSeries(seriesName, data.getSeries(seriesName));
+					if (data.getXValues(seriesName).size() < 200) {
+						ArrayList<String> onlyTimes = new ArrayList<>();
+						data.getXValues(seriesName).forEach( s -> onlyTimes.add(s.split("T")[1]));	
+						plotPanel.addSeries(seriesName, onlyTimes, data.getYValues(seriesName));
+					} else {
+						plotPanel.addSeries(seriesName, data.getXValues(seriesName), data.getYValues(seriesName));
+					}
```

Comment:
This piece of code has been referred to in a previous question. (Pointing it out in case there are any changes.)